apps/etterlatte-behandling/src/main/kotlin/oppgave/OppgaveService.kt

```diff
@@ -112,9 +91,15 @@ class OppgaveService(
 
         sikreAtOppgaveIkkeErAvsluttet(hentetOppgave)
         hentetOppgave.erAttestering() && sjekkOmkanTildeleAttestantOppgave()
+
         val eksisterendeSaksbehandler = hentetOppgave.saksbehandler?.ident
         if (eksisterendeSaksbehandler.isNullOrEmpty() || eksisterendeSaksbehandler == Fagsaksystem.EY.navn) {
             oppgaveDao.settNySaksbehandler(oppgaveId, saksbehandler)
+
+            // TODO: Fjerne dette. Midlertidig løsning for å støtte gammel flyt
+            if (hentetOppgave.status == Status.NY) {
+                oppgaveDao.endreStatusPaaOppgave(oppgaveId, Status.UNDER_BEHANDLING)
+            }
```

Comment:
Er disse midlertidige løsningene relevante nå med migreringen av oppgaver?

Reply:
Ingen anelse. Bare la det opp slik at det skal fungere tilsvarende gammel løsning. 

På sikt skal ikke tildeling medføre endring av status, men må bare være sånn enn så lenge for å sikre at jeg ikke brekker noe.

Reply:
Nei, tanken er at den skal forbli, men at vi ikke lenger skal sette en oppgave til status `under behandling` bare fordi en saksbehandler tildeles.